compose.yml

```diff
@@ -143,6 +146,34 @@ services:
     volumes:
       - ./backend/:/backend
 
+  statuser-dev:
+    env_file:
+      - backend.env
+    depends_on:
```

Comment:
I think it is better not to include worker and statuser as part of the backend-dev profile. 
If someone wants to use worker or statuser they can just add it to the Compose command.

Reply:
I see an issue, that the statuser and worker does not reload on changes even though the compile Daemon is used. can you check that please? @amirfefer

Reply:
I think it is related to the WORKDIR and the volume mapping.

Reply:
Yes, it is fixed now :)